pkg/kubernetes/kubernetes.go

```diff
@@ -7,42 +7,40 @@ package kubernetes
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
+	"path/filepath"
 	"runtime"
 	"time"
 
 	"github.com/dapr/cli/pkg/print"
-	cli_ver "github.com/dapr/cli/pkg/version"
-	"github.com/dapr/cli/utils"
+	v1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/helm/pkg/strvals"
 
 	"github.com/briandowns/spinner"
-	scheme "github.com/dapr/dapr/pkg/client/clientset/versioned"
-	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	helm "helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart"
+	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
 )
 
 const (
+	daprReleaseName   = "dapr"
+	daprHelmRepo      = "https://daprio.azurecr.io/helm/v1/repo"
```

Comment:
we should change helm repo to https://github.com/dapr/dapr-helm-charts.io

Reply:
jFYI - this is the correct url - https://dapr.github.io/dapr-helm-charts.io/ :)

Reply:
Done